Description: remove 3 quotes from the JSON file.
Author: Atlas Yu <atlas.yu@canonical.com>
Last-Update: 2022-09-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/quotes.json
+++ b/quotes.json
@@ -9,10 +9,8 @@
 {"text": "\"I have not failed. I've just found 10,000 ways that won't work.\"", "author": "Thomas A. Edison", "tags": ["edison", "failure", "inspirational", "paraphrased"]},
 {"text": "\"A woman is like a tea bag; you never know how strong it is until it's in hot water.\"", "author": "Eleanor Roosevelt", "tags": ["misattributed-eleanor-roosevelt"]},
 {"text": "\"A day without sunshine is like, you know, night.\"", "author": "Steve Martin", "tags": ["humor", "obvious", "simile"]},
-{"text": "\"This life is what you make it. No matter what, you're going to mess up sometimes, it's a universal truth. But the good part is you get to decide how you're going to mess it up. Girls will be your friends - they'll act like it anyway. But just remember, some come, some go. The ones that stay with you through everything - they're your true best friends. Don't let go of them. Also remember, sisters make the best friends in the world. As for lovers, well, they'll come and go too. And baby, I hate to say it, most of them - actually pretty much all of them are going to break your heart, but you can't give up because if you give up, you'll never find your soulmate. You'll never find that half who makes you whole and that goes for everything. Just because you fail once, doesn't mean you're gonna fail at everything. Keep trying, hold on, and always, always, always believe in yourself, because if you don't, then who will, sweetie? So keep your head high, keep your chin up, and most importantly, keep smiling, because life's a beautiful thing and there's so much to smile about.\"", "author": "Marilyn Monroe", "tags": ["friends", "heartbreak", "inspirational", "life", "love", "sisters"]},
 {"text": "\"It takes a great deal of bravery to stand up to our enemies, but just as much to stand up to our friends.\"", "author": "J.K. Rowling", "tags": ["courage", "friends"]},
 {"text": "\"If you can't explain it to a six year old, you don't understand it yourself.\"", "author": "Albert Einstein", "tags": ["simplicity", "understand"]},
-{"text": "\"You may not be her first, her last, or her only. She loved before she may love again. But if she loves you now, what else matters? She's not perfect\u2014you aren't either, and the two of you may never be perfect together but if she can make you laugh, cause you to think twice, and admit to being human and making mistakes, hold onto her and give her the most you can. She may not be thinking about you every second of the day, but she will give you a part of her that she knows you can break\u2014her heart. So don't hurt her, don't change her, don't analyze and don't expect more than she can give. Smile when she makes you happy, let her know when she makes you mad, and miss her when she's not there.\"", "author": "Bob Marley", "tags": ["love"]},
 {"text": "\"I like nonsense, it wakes up the brain cells. Fantasy is a necessary ingredient in living.\"", "author": "Dr. Seuss", "tags": ["fantasy"]},
 {"text": "\"I may not have gone where I intended to go, but I think I have ended up where I needed to be.\"", "author": "Douglas Adams", "tags": ["life", "navigation"]},
 {"text": "\"The opposite of love is not hate, it's indifference. The opposite of art is not ugliness, it's indifference. The opposite of faith is not heresy, it's indifference. And the opposite of life is not death, it's indifference.\"", "author": "Elie Wiesel", "tags": ["activism", "apathy", "hate", "indifference", "inspirational", "love", "opposite", "philosophy"]},
@@ -62,7 +60,6 @@
 {"text": "\"That's the problem with drinking, I thought, as I poured myself a drink. If something bad happens you drink in an attempt to forget; if something good happens you drink in order to celebrate; and if nothing happens you drink to make something happen.\"", "author": "Charles Bukowski", "tags": ["alcohol"]},
 {"text": "\"You don't forget the face of the person who was your last hope.\"", "author": "Suzanne Collins", "tags": ["the-hunger-games"]},
 {"text": "\"Remember, we're madly in love, so it's all right to kiss me anytime you feel like it.\"", "author": "Suzanne Collins", "tags": ["humor"]},
-{"text": "\"To love at all is to be vulnerable. Love anything and your heart will be wrung and possibly broken. If you want to make sure of keeping it intact you must give it to no one, not even an animal. Wrap it carefully round with hobbies and little luxuries; avoid all entanglements. Lock it up safe in the casket or coffin of your selfishness. But in that casket, safe, dark, motionless, airless, it will change. It will not be broken; it will become unbreakable, impenetrable, irredeemable. To love is to be vulnerable.\"", "author": "C.S. Lewis", "tags": ["love"]},
 {"text": "\"Not all those who wander are lost.\"", "author": "J.R.R. Tolkien", "tags": ["bilbo", "journey", "lost", "quest", "travel", "wander"]},
 {"text": "\"Do not pity the dead, Harry. Pity the living, and, above all those who live without love.\"", "author": "J.K. Rowling", "tags": ["live-death-love"]},
 {"text": "\"There is nothing to writing. All you do is sit down at a typewriter and bleed.\"", "author": "Ernest Hemingway", "tags": ["good", "writing"]},
@@ -99,4 +96,4 @@
 {"text": "\"Never tell the truth to people who are not worthy of it.\"", "author": "Mark Twain", "tags": ["truth"]},
 {"text": "\"A person's a person, no matter how small.\"", "author": "Dr. Seuss", "tags": ["inspirational"]},
 {"text": "\"... a mind needs books as a sword needs a whetstone, if it is to keep its edge.\"", "author": "George R.R. Martin", "tags": ["books", "mind"]}
-]
\ No newline at end of file
+]
